src/ui/src/core_components/content/CoreAnnotatedText.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+	<div class="AnnotatedText">
+		<span v-for="(content, i) in fields.text.value" :key="content + i">
+			<template v-if="typeof content === 'string'">{{
+				content
+			}}</template>
+			<span
+				v-if="Array.isArray(content)"
+				class="annotation"
+				:style="{ background: content[2] || generateColor(content[1]) }"
+			>
+				{{ content[0] }}
+				<span v-if="content[1]" class="annotation-subject">
+					{{ content[1] }}
+				</span>
+			</span>
+		</span>
+		<template v-if="fields.copyButtons.value === 'yes'">
+			<BaseControlBar
+				:copy-raw-content="textToString(fields.text.value)"
+				:copy-structured-content="stringifyData(fields.text.value)"
+			/>
+		</template>
+	</div>
+</template>
+
+<script lang="ts">
+import { cssClasses, primaryTextColor } from "../renderer/sharedStyleFields";
+export default {
+	writer: {
+		name: "Annotated text",
+		description: "Shows text with annotations",
+		category: "Content",
+
+		fields: {
+			text: {
+				name: "KeyValue",
+				type: FieldType.Object,
+				desc: "Value array with text/annotations. Must be a JSON string or a state reference to an array.",
```

Comment:
It needs to be a an array, no ?
```suggestion
				desc: "Value array with text/annotations. Must be a JSON Array as string or a state reference to an array.",
```